test_flake8_graphql.py

```diff
@@ -59,7 +59,7 @@ def my_function():
     """)
     result = flake8dir.run_flake8()
     assert result.out_lines == [
-        './example.py:6:12: GQL100: Syntax Error GraphQL (4:9) Expected {, found Name "foobars"',
+        './example.py:14:0: GQL100: Syntax Error GraphQL (4:9) Expected {, found Name "foobars"',
```

Comment:
the last character of the doc string.

It seems you get the first character in regular strings and the last character in doc strings